Formula/seqtk.rb

```diff
@@ -0,0 +1,21 @@
+class Seqtk < Formula
+  desc "Toolkit for processing sequences in FASTA/Q formats"
+  homepage "https://github.com/lh3/seqtk"
+  url "https://github.com/lh3/seqtk/archive/v1.2.tar.gz"
+  sha256 "bd53316645ab10f0aaba59e1e72c28442ee4c9c37fddaacce5e24757eff78d7b"
+  head "https://github.com/lh3/seqtk.git"
+
+  def install
+    system "make"
+    bin.install "seqtk"
```

Comment:
Have you requested a `make install` target?

Reply:
Done: https://github.com/lh3/seqtk/issues/103

Reply:
needs to be merged upstream.